src/locale/es/LC_MESSAGES/django.po

```diff
@@ -4106,3 +4106,24 @@ msgstr "Actualizar consejo de prudencia"
 
 msgid "Update the Danger indication"
 msgstr "Actualizar indicacion de peligro"
+
+msgid "Laboratory Room not belongs to this laboratory"
+msgstr "La sala de laboratorio no pertenece al laboratorio"
+
+msgid "Laboratory Room does not belongs to this organization"
+msgstr "La sala de laboratorio no pertenece a la organización"
+
+msgid "Shelfobject does not belongs to this laboratory."
+msgstr "El objecto de estante no pertenece al laboratorio."
+
+msgid "Shelfobject does not belongs to this organization."
```

Comment:
When the auxiliary "does" is used then the verb doesn't change, meaning it doesn't need to have the s for the third person.  For example, instead of "Shelfobject does not belongs to this organization." it should be "Shelfobject does not **belong** to this organization."